plugins/webhooks/github.js

```diff
@@ -38,6 +38,7 @@ function pullRequestOpened(options, request, reply) {
     const buildFactory = request.server.app.buildFactory;
     const eventId = options.eventId;
     const pipelineId = options.pipelineId;
+    const jobId = options.jobId;
```

Comment:
`jobId` is just the hash of `pipelineId` and `name`. Since we already generate here: https://github.com/screwdriver-cd/screwdriver/blob/da6cf99d835bc8750fe4e6e6792f1b176cd944d8/plugins/webhooks/github.js#L234

we might as well pass it in?